subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformIntegrationTest.groovy

```diff
@@ -2352,6 +2352,7 @@ Found the following transforms:
             ${declareTransform(transformImplementation)}
 
             task resolve(type: Copy) {
+                duplicatesStrategy = 'INCLUDE'
```

Comment:
Why is this needed?

Reply:
Since we sometimes resolve two `jar`s with the same name and copy them into the same directory.

Reply:
Isn't that something we should fix?

Reply:
I don't think so. The point of the task is to resolve the configurations involved, the tests don't care where the artifacts are copied to.